app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt

```diff
@@ -155,8 +161,27 @@ class ProfileActivity : AppCompatActivity() {
         }
     }
 
+    private fun setImage(imageURI: Uri) {
+        CoroutineScope(Dispatchers.Main).launch {
+            val task = async(Dispatchers.IO) {
+                try {
+                    val bit = BitmapGetterApi.download(imageURI.toString())
+                    bit.get(10, TimeUnit.SECONDS)
+                }catch (e: Exception){
+                    null

```

Comment:
why the handle exception is null?

Reply:
@MaximeZmt wrote this function I just reused it, so maybe he can answer better :)

Reply:
It as was null because it was not a critical feature. I mean that the download of an image can fail for many reasons (user, server, Internet Provider, other users, ...)
Therefore if it does not works, As it was implemented at the beginning of the semester I've put null not knowing how to handle it.
Maybe there is a better way to do it now.